cli/types/cypress.d.ts

```diff
@@ -2876,7 +2877,7 @@ declare namespace Cypress {
      * To enable test retries only in runMode, set e.g. `{ openMode: null, runMode: 2 }`
      * @default null
      */
-    retries: Nullable<number | { runMode?: Nullable<number>, openMode?: Nullable<number> }>
+    retries: Cypress.Nullable<number | { runMode?: Cypress.Nullable<number>, openMode?: Cypress.Nullable<number> }>
```

Comment:
honestly, this isn't hardly used. IMO it'd be similar to remove it all together:

```suggestion
    retries: number | { runMode?: number | null, openMode?: number | null } | null
```

Reply:
I am not a cypress developer and do not have voting rights. but I came here just because in our project we came to a consensus that the Nullable construction is redundant and does not add readability. Right now I'm cleaning up the code.

Should I remove this type here as well? Or does it require further discussion?